config/havp/havp.inc

```diff
@@ -76,7 +76,7 @@ define('HVDEF_PROXYPORT',	            '8080');
 define('HVDEF_MAXSCANSIZE',             '5000000'); # [bytes] ! do not enter 0 or big size !
 define('HVDEF_MAXARCSCANSIZE',          '5000000'); # [bytes] ! do not enter 0 or big size !
 define('HVDEF_PID_FILE',                '/var/run/havp.pid');
-define('HVDEF_WORK_DIR',                '/usr/local/etc/havp');
+define('HVDEF_WORK_DIR',                '/usr/pbi/havp-amd64/local/etc');
```

Comment:
You need to check pfSense version and also system arch (amd64 or i386) before set it. Take a look at an example at https://github.com/pfsense/pfsense-packages/blob/master/config/snort/snort_defs.inc#L38